src/main/resources/containers.json

```diff
@@ -1250,8 +1250,6 @@
     ]
   },
   {
-    "artifact_id": "glassfish",
-    "group_id": "org.glassfish.main.distributions",
     "name": "Arquillian Container GlassFish Managed",
```

Comment:
Thanks for the explanation. I'm just wondering if using defaults (so not really defining any config entry in the `arquillian.xml`) would make sense. Are we letting user to define e.g. which port to use for managed container?